manifests/postgresql_settings.pp

```diff
@@ -125,11 +124,12 @@
       db                    => 'pe-puppetdb',
       table_name            => 'reports',
       table_attribute       => 'autovacuum_vacuum_scale_factor',
-      table_attribute_value => "${reports_autovacuum_vacuum_scale_factor}",
+      table_attribute_value => String($reports_autovacuum_vacuum_scale_factor),
     }
   }
 
   if $manage_reports_autovacuum_cost_delay {
     pe_databases::set_puppetdb_table_autovacuum_cost_delay_zero { 'reports' : }
   }
 }
+
```

Comment:
https://docs.puppet.com/puppet/5.0/style_guide.html#spacing-indentation-and-whitespace

> Must end last line with a new line